tests/test_backend_loki_fields.py

```diff
@@ -0,0 +1,82 @@
+from typing import Any, Dict
+import pytest
+from sigma.backends.loki import LogQLBackend
+from sigma.collection import SigmaCollection
+from sigma.exceptions import SigmaFeatureNotSupportedByBackendError, SigmaTypeError
+from sigma.modifiers import modifier_mapping
+
+
+@pytest.fixture
+def loki_backend() -> LogQLBackend:
+    return LogQLBackend()
+
+
+# Mapping from modifier identifier strings to modifier classes
+modifier_sample_data: Dict[str, Any] = {
```

Comment:
Could we change this to a `Dict[str, tuple[Any, str]]` adding the modifier and then the expected input and output

Reply:
Addressed.